app/main.ipynb

```diff
@@ -100,18 +125,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "arr_np_float = np.array([0, 0.5, 1.0, 1.5, 2.0, 2.5, 3.0, 3.5], dtype=np.float16)\n",
+    "arr_np_float[[0, 2, -1]]"
```

Comment:
The array creation and multi-indexing to access specific elements are correctly implemented according to the task requirements.